File: Demos/Datasnap/Server/proxy/java_blackberry/com/embarcadero/javablackberry/TJSONFalse.java
//*******************************************************
//
//               Delphi DataSnap Framework
//
// Copyright(c) 1995-2011 Embarcadero Technologies, Inc.
//
//*******************************************************

package com.embarcadero.javablackberry;

/**
 * 
 * Contains and returns the boolean value false like a JSON .
 * 
 */

public class TJSONFalse extends TJSONValue {

	public int getJsonValueType() {
		return JSONValueType.JSONFalse;
	}

	/**
	 * Returns the String "false"
	 * 
	 * @return
	 */
	public String asJSONString() {
		return "false";
	}

	public Object getInternalObject() {
		return Boolean.FALSE;
	}

	public String toString() {
		return asJSONString();
	}

}

File: Demos/Datasnap/Server/proxy/objectivec_ios81/Proxy/TJSONFalse.m
//*******************************************************
//
//               Delphi DataSnap Framework
//
// Copyright(c) 1995-2011 Embarcadero Technologies, Inc.
//
//*******************************************************

#import "TJSONFalse.h"


@implementation TJSONFalse
-(NSString *) asJSONString{
	return @"false";
}
-(id) getInternalObject {
		return [NSNumber numberWithBool:FALSE];
}
-(NSString *) toString{
	return [self asJSONString];
}
-(JSONValueType) getJSONValueType{
	return JSONFalse;
}

@end
